test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -179,7 +179,7 @@ public async void DependencyGraphRestoreUtility_LegacyPackageRef_Restore_Generat
                         _threadingService);
 
                     var projectPathA = Path.Combine(testSolutionManager.TestDirectory, "ProjectA");
-                    var fullProjectPathA = Path.Combine(projectPathA, "project1.csproj");
+                    var fullProjectPathA = Path.Combine(projectPathA, "Project1.csproj");
```

Comment:
why change this?

Reply:
extra test to ensure that the lock file lower cases project names.